slices/grep.yaml

```diff
@@ -13,6 +13,7 @@ slices:
     # See: https://manpages.ubuntu.com/manpages/jammy/en/man1/grep.1.html
     essential:
       - bash_bins
```

Comment:
same as https://github.com/canonical/chisel-releases/pull/118#discussion_r1480907936

Reply:
Cool, I can do that. Didn't realize Debian / Ubuntu used Dash as the default for that ; when I used to do RHEL-like packaging it was always bash.